templates/systemd/system.service.tpl

```diff
@@ -21,6 +21,9 @@ LimitSTACK=10485760
 
 User={{.User}}
 ExecStart={{.DeployDir}}/scripts/run_{{.ServiceName}}.sh
+{{- if eq .ServiceName "prometheus"}}
+ExecReload=/bin/kill -HUP $MAINPID
```

Comment:
It's `/usr/bin/kill` on CentOS 7, openSUSE Tumbleweed and ArchLinux, but `/bin/kill` on Debian Stretch and Buster.

Reply:
OK seems they all have `/bin/kill` as an alias